varia/inc/template-tags.php

```diff
@@ -212,16 +211,16 @@ function varia_post_thumbnail() {
 		if ( is_singular() ) :
 			?>
 
-			<figure class="post-thumbnail">
+			<figure class="post-thumbnail responsive-max-width">
```

Comment:
@allancole I remember the feedback saying the featured image can be too big if it goes full width. But adding this css class now changes a lot of sites, and the users can't change it back. Are we ok with that?

Reply:
Ahhh, good catch! I was thinking it should have just been the `.aligndefault` column width from the beginning so I just changed it. But you’re right, if we add that class now it‘ll disrupt things for customers. I’ll remove it…